python/plugins/processing/tests/ModelerTest.py

```diff
@@ -66,6 +66,15 @@ def testModelerParametersDialogAvailableValuesOfType(self):
         self.assertEqual(set(p.name for p in dlg.getAvailableValuesOfType([ParameterString, ParameterNumber, ParameterFile])),
                          set(['string', 'string2', 'number', 'file']))
 
+        a = Algorithm("qgis:clip")
```

Comment:
This should be in it's own function, not at the end of testModelerParametersDialogAvailableValuesOfType ;)